### PR TITLE
Blog posit

### PR DESCRIPTION
**Pull request by jules32**
Hi @stefaniebutland , 

Could you please review and copy-edit and post this blog? We can confirm the date in https://github.com/Openscapes/how_we_work/issues/285.

As you review there are some specific decisions and items to address: 

- [x] Ok with the order of the two main sections, or should Documenting things be first? 
- [x] Please add any text to help the overview or transitions, and if any of the thoughts are too incomplete could you improve them
- [x] please add alt text and figure captions to each image
- [x] please add any links you think are good throughout (ie RLadies, rOpenSci, etc) (there aren't links to slides from the web portal)

Also: if you'd like to add based on watching talks, please do and co-author!

### CODE REVIEW

File: blog/2023-09-22-documenting-things-posit-conf/index.qmd [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
... Fred Hutch Data Science Lab. His talk was about the [Tidy Data Tutor](https://tidydatatutor.com/) that he developed to help visualize tidyverse operations. He framed this in a bigger idea that **code is data**, which is something I'd never heard about before. Also, this paper [Tools for Analyzing R Code the Tidy Way](https://journal.r-project.org/articles/RJ-2020-011/) by Sean, Jeff Leek, and Lucy D’Agostino McGowan is now on my list to read! Sean Kross is such a hero of mine, one of the early friendly people that made me feel like I belonged in the R community, and I'm so thrilled that we've been collaborating for nearly 5 years. He was part of our very first Champions Cohort, when he was exploring studying Openscapes as part of his PhD. He has also built our [kyber R package](https://openscapes.github.io/kyber) that automates a lot of our infrastructure to lead Champions Cohorts. And, now we're leading the very first Champions Cohort at Fred Hutch, together with Stefanie Butland and Sean's colleague (and R Lady) Monica Gerber. It was wonderful to see Sean and Monica in person! Sean also gave gave me a [swirl](https://swirlstats.com/) sticker which I have coveted for a long time.
```
Comment (jules32) [inline blog/2023-09-22-documenting-things-posit-conf/index.qmd:87]:
```suggestion
Sean Kross is a Data Staff Scientist at the Fred Hutch Data Science Lab. His talk was about the [Tidy Data Tutor](https://tidydatatutor.com/) that he developed to help visualize tidyverse operations. He framed this in a bigger idea that **code is data**, which is something I'd never heard about before. Holy Moly Sean, Tidy Data Tutor has 50K users across 150 countries! And this growth is over maybe 2 years? Amazing. And speaks to its usefulness. Also, this paper [Tools for Analyzing R Code the Tidy Way](https://journal.r-project.org/articles/RJ-2020-011/) by Sean, Jeff Leek, and Lucy D’Agostino McGowan is now on my list to read! Sean Kross is such a hero of mine, one of the early friendly people that made me feel like I belonged in the R community, and I'm so thrilled that we've been collaborating for nearly 5 years. He was part of our very first Champions Cohort, when he was exploring studying Openscapes as part of his PhD. He has also built our [kyber R package](https://openscapes.github.io/kyber) that automates a lot of our infrastructure to lead Champions Cohorts. And, now we're leading the very first Champions Cohort at Fred Hutch, together with Stefanie Butland and Sean's colleague (and R Lady) Monica Gerber. It was wonderful to see Sean and Monica in person! Sean also gave gave me a [swirl](https://swirlstats.com/) sticker which I have coveted for a long time. 
```

File: blog/2023-09-22-documenting-things-posit-conf/index.qmd [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+#image: 
+---
+
+*Last week I've been at #PositConf2023 in Chicago, reconnecting with the open data science community through folks from RLadies, Posit, and the greater #rstats and python community. I gave a talk, sharing about documenting things and how intentional approaches is helping teams tackle hard challenges and change organizational culture. This post is a light debrief of highlights from the Conference, as well as a short overview of my talk.*
+
+*Quick links:*
+
+-   [*Posit Conf YouTube*]() - coming soon with all talks from the Conference
+- [Documenting things: openly for future us](https://openscapes.github.io/documenting-things) - my slides
+
```
Comment (jules32) [inline blog/2023-09-22-documenting-things-posit-conf/index.qmd:19]: - [Blog summary of the conference by Millie Symns!](https://milliesymns.com/blog/2023-09-25-spoke-at-posit-conf-2023/)

Comment (jules32) [reply blog/2023-09-22-documenting-things-posit-conf/index.qmd:19]: Just read this and - Ah I had meant to make that a commit not a comment, but if you could please add @stefaniebutland, thank you!

**Comment by stefaniebutland** [comment]
added a closing sentence
> Together with this awesome community we are building technical and social infrastructure that we can leverage to shift institutional culture and develop climate solutions with Open Science!

**Comment by stefaniebutland** [comment]
Published post via [new PR](https://github.com/Openscapes/website-new/pull/60), new branch since I ran into merge conflicts from .json & .html files

**Comment by stefaniebutland** [comment]
Published post via [new PR](https://github.com/Openscapes/website-new/pull/60), new branch since I ran into merge conflicts from .json & .html files

**Pull request closed without merging**